js/gantt-helper.js

```diff
@@ -29,13 +29,15 @@
 /* global gantt */
 /* global displayAjaxMessageAfterRedirect */
 
-var GlpiGantt = (function() {
-
-    var readonly = false;
-    var plugin_path = CFG_GLPI.root_doc + '/' + (GLPI_PLUGINS_PATH['gantt'] ?? "");
-    var url = plugin_path +  '/ajax/gantt.php';
-    var parseDateFormat = "%Y-%m-%d %H:%i";
-    var uiDateFormat = null;
+//TODO: This should be a class
+const GlpiGantt = (function() {
+
+    //TODO: Why is this always false?
+    const readonly = false;
```

Comment:
is it used at all ?

Reply:
It is. It seems like everything was implemented to allow a readonly view, but the value was just set to false.